fortigate-autoscale/fortigate-bootstrap-config-strategy.ts

```diff
@@ -272,23 +272,40 @@ export class FortiGateBootstrapConfigStrategy implements BootstrapConfigurationS
             config = this.processConfigV2(config, sourceData);
         }
 
-        const psksecret = this.settings.get(FortiGateAutoscaleSetting.FortiGatePskSecret).value;
+        const psksecret = this.normalizeFOSCmdInput(
```

Comment:
maybe you could just make a copy of FortiGateAutoscaleSettings where everything is normalized?

`normalizedSettings = Object.entries(FortiGateAutoscaleSettings).reduce((r, [key, value]) => { r[key] = this.normalizeFOSCmdInput(value); return r}, {})`

Reply:
I'll make an improvement for it.